hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperationTestUtil.java

```diff
@@ -47,4 +49,9 @@ public static void addAbfsHttpOpProcessResponseMock(final AbfsRestOperation spie
   public static void setResult(final AbfsRestOperation op, final AbfsHttpOperation result) {
     op.setResult(result);
   }
+
+  public static void assertTracingContextInstantiationCount(final AbfsClient client, final int numberOfInstantiation) {
```

Comment:
This method is not used anywhere ?

Reply:
Yes,
It is not used anywhere...
Removing it.